docs/guide/Titanium_SDK/Titanium_SDK_Guide/Atom_Package/Atom_Package_Release_Notes/README.md

```diff
@@ -11,7 +11,7 @@ Atom Package 1.1.0 is a minor release that includes new features, improvements,
 
 ### New features
 
-* [ATOM-11](https://jira.appcelerator.org/browse/ATOM-11) - Snippets
+* [ATOM-11](https://jira-archive.titaniumsdk.com/ATOM-11) - Snippets
```

Comment:
Same in this file... ATOM does not exist, remove link

Reply:
links removed